dataactvalidator/config/sqlrules/sqlRules.csv

```diff
@@ -120,3 +120,7 @@ D13,"LegalEntityZIP5 is required for domestic recipients (i.e., when LegalEntity
 D14,"LegalEntityZIPLast4 must be blank for aggregate records (i.e., when RecordType = 1)","LegalEntityZIPLast4 must be blank for aggregate records (i.e., when RecordType = 1)",n,detached_award,fatal,,d14_detached_award_financial_assistance_1
 D14,"LegalEntityZIPLast4 must be blank for foreign recipients (i.e., when LegalEntityCountryCode is not USA)","LegalEntityZIPLast4 must be blank for foreign recipients (i.e., when LegalEntityCountryCode is not USA)",n,detached_award,fatal,,d14_detached_award_financial_assistance_2
 D14,"LegalEntityZIPLast4 is required for domestic recipients (i.e., when LegalEntityCountryCode = USA) for non-aggregate records (i.e., when RecordType = 2)","LegalEntityZIPLast4 is required for domestic recipients (i.e., when LegalEntityCountryCode = USA) for non-aggregate records (i.e., when RecordType = 2)",n,detached_award,fatal,,d14_detached_award_financial_assistance_3
+D15,"LegalEntityForeignCityName is required for foreign recipients (i.e., when LegalEntityCountryCode is not USA)","LegalEntityForeignCityName is required for foreign recipients (i.e., when LegalEntityCountryCode is not USA)",n,detached_award,fatal,,d15_detached_award_financial_assistance_1
+D15,"LegalEntityForeignCityName must be blank for domestic recipients (i.e., when LegalEntityCountryCode = USA)","LegalEntityForeignCityName must be blank for domestic recipients (i.e., when LegalEntityCountryCode = USA)",n,detached_award,fatal,,d15_detached_award_financial_assistance_2
+D16,"LegalEntityForeignProvinceName is optional for foreign recipients (i.e., when LegalEntityCountryCode is not USA). LegalEntityForeignProvinceName must be blank for domestic recipients (i.e., when LegalEntityCountryCode = USA)","LegalEntityForeignProvinceName is optional for foreign recipients (i.e., when LegalEntityCountryCode is not USA). LegalEntityForeignProvinceName must be blank for domestic recipients (i.e., when LegalEntityCountryCode = USA)",n,detached_award,fatal,,d16_detached_award_financial_assistance

```

Comment:
Since it's "optional" for foreign recipients the foreign recipient will never be the cause of the error so it really doesn't need to be in the error text. You only need to list the part that can actually fail (the sentence about domestic recipients)

This comment is for both D16 and D17